examples/error-cause/error_cause.md

```diff
@@ -0,0 +1,7 @@
+# Error cause
+
+Ballerina facilitates in having a cause in an error value. It can be passed as a positional argument to the error constructor. The cause argument is optional in the error constructor.
```

Comment:
```suggestion
Ballerina facilitates having a cause in an error value. It can be passed as a positional argument to the error constructor. The cause argument is optional in the error constructor.
```